src/test/run-make-fulldeps/hotplug_codegen_backend/the_backend.rs

```diff
@@ -66,7 +67,7 @@ impl CodegenBackend for TheBackend {
         _metadata: EncodedMetadata,
         _need_metadata_module: bool,
     ) -> Box<dyn Any> {
-        use rustc_hir::def_id::LOCAL_CRATE;
```

Comment:
My editor auto-formatted this file according to the project `rustfmt.toml` when I edited this. This file seems to be currently ignored by `./x.py fmt`, but I didn't see the harm in keeping these changes.